File: ProgrammingLanguage/Python/HelloWorld/Fac.py
#! /usr/bin/env python

def fac(n) :
    sum = 1
    while n :
        sum *= n
        n -= 1
    return sum

def main() :
    for i in xrange(10) :
        print i, fac(i)

if __name__ == "__main__" : main()

File: ProgrammingLanguage/Java/HelloWorld/Fac.java
public class Fac 
{
    public static long fac(long n) {
        if (n == 0) return 1;
        return n*fac(n-1);
    }

    public static void main(String[] args) {
        for (long i = 0; i < 21; ++i) {
            System.out.println(i + " " + fac(i));
        }
    }
}
